test/Main.hs

```diff
@@ -23,5 +25,7 @@ allTests =
       mutHMTests,
       mutSetTests,
       destArrayTests,
-      polarizedArrayTests
+      polarizedArrayTests,
+      vInspectionTests,
+      replicatorInspectionTests
```

Comment:
Divide this list in two `testGroup`s one for data structure tests, and one for inspection tests.